docs/ref/models/querysets.txt

```diff
@@ -3604,6 +3618,10 @@ by the aggregate.
         This is the SQL equivalent of ``AVG(DISTINCT <field>)``. The default
         value is ``False``.
 
+    .. versionchanged:: 4.0
+
+        The ``default`` argument was added.
+
```

Comment:
IMO we don't need `versionchanged` annotations in every aggregate. I'd leave in only in [`Aggregate`](https://docs.djangoproject.com/en/3.2/ref/models/expressions/#django.db.models.Aggregate) docs (we did the same when adding `filter`, see b78d100fa62cd4fbbc70f2bae77c192cb36c1ccd).
```suggestion
```